### DISCUSSION TITLE
he daily notional value limit for transfers on Klaytn has been exceeded

### DISCUSSION THREAD

**Discussion by si2you** [wormhole-foundation/wormhole#1517] [Problems / bugs] [answered]
Hello, 
Hello, I bridged DAI from Klaytn network to Etherium chain. But I got message " The daily notional value limit for transfers on Klaytn has been exceeded. As a result, the VAA for this transfer is pending.  What should I do? I can redeem after 24 hours?

**Comment by evan-gray** [comment, marked as answer]
Then transactions are held due to exceeding the notional limit, the maximum wait time is 24 hours (as of this writing). For more details on the governor, see the [whitepaper](https://github.com/wormhole-foundation/wormhole/blob/dev.v2/whitepapers/0007_governor.md).
